src/contexts/bridge.context.tsx

```diff
@@ -630,10 +633,12 @@ const BridgeProvider: FC<PropsWithChildren> = (props) => {
             pendingTx.type === "deposit" &&
             isPendingDepositInApiBridges(pendingTx.depositTxHash)
           ) {
-            return storage.removePendingTx(env, pendingTx.depositTxHash);
-          }
-          if (pendingTx.type === "claim" && isPendingClaimInApiBridges(pendingTx.claimTxHash)) {
-            return storage.removePendingTx(env, pendingTx.depositTxHash);
+            storage.removePendingTx(account.data, env, pendingTx.depositTxHash);
+          } else if (
```

Comment:
Can we go back to the single `if ... else` statements? `if ... else ... if` without a final `else` condition is harder to follow